README.rst

```diff
@@ -117,7 +117,7 @@ Call ``make test``.
 Publishing a Release
 --------------------
 
-After a PR merges, a new version of the package will automatically be released by Travis when the commit is tagged. Use::
+After a PR merges, a new version of the package will automatically be released by Github workflow when a release is created. Use::
```

Comment:
The text below says specifically not to create a GitHub release, we should update that also.  And I'd specify "...when a GitHub release is created..." on this line, maybe even with a link to the releases page.  Otherwise it looks like you just need to create and push the tag, given the instructions below.